docs/usage/self-hosted-configuration.md

```diff
@@ -63,6 +63,41 @@ But before you disable templating completely, try the `allowedPostUpgradeCommand
 
 ## allowScripts
 
+## allowedHeader
+
+This is option particularly useful when a registry employs a specific authentication system not already covered by Renovate's standard credential handling in `hostRules`.
+By default, all header starting with "X-" are allowed, but you can permit additional header using this option.
+If declared, it will override the default "X-" allowed header, so you should include them in your config if you wish for them to remain allowed.
```

Comment:
```suggestion
Any set `allowedHeader` overrides the default "X-" allowed header, so you should include them in your config if you wish for them to remain allowed.
```

It's not clear to me what `included them` refers to. Overall this sentence is tough to understand.